src/main/java/org/skife/jdbi/v2/Update.java

```diff
@@ -66,6 +66,26 @@ public Integer munge(Statement results) throws SQLException
         }
     }
 
+    /**
+     * Execute the statement for a large number of affected rows
+     * @return the number of rows modified
+     */
+    public long executeLarge()
```

Comment:
Why not `executeLong` or `executeReturningLong`? `Large` is not a clear name.

Reply:
I was just tracking the underlying *Large* naming from java.sql.Statement.  I can change to Long though, you're right.